src/components/learning-guide.cjsx

```diff
@@ -20,6 +22,10 @@ LearningGuide = React.createClass
   propTypes:
     courseId: React.PropTypes.string.isRequired
 
+  getInitialState: ->
+    period_id: "1"
```

Comment:
Is this a safe default (the same assumption had issues for the Learning Guide when viewing a 2nd course)?